src/codeu/chat/ClientMain.java

```diff
@@ -34,6 +29,61 @@
   private static Time lastLogBackup;
   private static final long BACKUP_RATE_IN_MS = 30000;
 
+  private static Chat chat;
+
+  private static void reloadOldInterests() throws IOException {
+    // Open the transaction log file for reading
+    FileReader fileReader = new FileReader("data/transaction_log.txt");
+    BufferedReader bufferedReader = new BufferedReader(fileReader);
```

Comment:
Fixed, since there'll probably be multiple people working on this pull request I'll note what I've done so nothing will be done twice.